scripts/kani-regression.sh

```diff
@@ -74,7 +74,8 @@ for testp in "${TESTS[@]}"; do
   suite=${testl[0]}
   mode=${testl[1]}
   echo "Check compiletest suite=$suite mode=$mode"
-  cargo run -p compiletest --quiet -- --suite $suite --mode $mode --quiet
+  cargo run -p compiletest --quiet -- --suite $suite --mode $mode \
+      --quiet --no-fail-fast
```

Comment:
Sorry if I didn't make myself clear earlier, but some people are actually annoyed if they have to wait for all tests in a suite to complete before they're able to see what went wrong.

Reply:
I guess there are two scenarios here:
 * Developers running the regression: They want it to fail fast so they don't have to wait for the suite to complete before seeing the errors they got.
 * CI running the regression: Developers want it to NOT fail fast so they get more comprehensive results.

Reply:
Discussed offline. We'll keep the script simple for now and keep the default as it is meant to be run in CI (with `--no-fail-fast`). We can customize the script later or developers can individually run `compiletest` separately.